Explorer/Assets/DCL/Character/CharacterMotion/Animation/AvatarAnimationEventsHandler.cs

```diff
@@ -105,6 +112,8 @@ private void PlayStepSoundForFoot(Transform footTransform)
                 case MovementKind.Run:
                     if (currentTime - lastFootstepTime > runIntervalSeconds)
                     {
+                        PlayerStepped?.Invoke();
+
```

Comment:
These repeating blocks look like a great candidate to extract a method, what do you thing?
Something like

```
HandleStepSound(float intervalSeconds, AvatarAudioClipType audioClipType) { ... }
```
and then it's called like:

```
case MovementKind.Walk:
   HandleStepSound(walkIntervalSeconds, AvatarAudioSettings.AvatarAudioClipType.StepWalk);
case MovementKind.Jog:
   HandleStepSound(jogIntervalSeconds, AvatarAudioSettings.AvatarAudioClipType.StepJog);
   break;
case MovementKind.Run:
   HandleStepSound(runIntervalSeconds, AvatarAudioSettings.AvatarAudioClipType.StepRun);
   break;
```

Reply:
sure, I like you DRY vision ;)

Reply:
I took it to a bit bigger scope, hope you'll enjoy it 🙂

Reply:
It's art ❤️